### ISSUE TITLE
Question about running transe

### ISSUE THREAD

**Issue by shaoyx** [mana-ysh/knowledge-graph-embeddings#5]
Hi all,
I ran TransE on WordNet with following command:
`python train.py --mode pairwise --ent ../dat/wordnet-mlj12/train.entlist --rel ../dat/wordnet-mlj12/train.rellist --train ../dat/wordnet-mlj12/wordnet-mlj12-train.txt --valid ../dat/wordnet-mlj12/wordnet-mlj12-valid.txt --method transe`
But the loss is not converging. 
![image](https://user-images.githubusercontent.com/2384668/34995154-9b8c7126-fb10-11e7-8f16-ea2e7a15ea02.png)

Do I have set the wrong parameters?

**Comment by mana-ysh** [comment]
Hi @simon0227 

Thank you for reporting.
It seems that it's my mistake. I'll check and fix my code.

**Comment by mana-ysh** [comment]
This is due to the incorrect score calculation and I fixed it. 
Please retry to run TransE. it should work and converge the training loss.

**Comment by shaoyx** [comment]
It works now. Thanks~
